lib/ae_mdw/node.ex

```diff
@@ -116,6 +116,10 @@ defmodule AeMdw.Node do
   end
 
   @spec tx_ids(atom()) :: %{atom() => non_neg_integer()}
+  def tx_ids(:spend_tx) do
+    %{sender_id: 0, recipient_id: 0}
+  end
+
```

Comment:
This is going to be overriden by the `SmartGlobal.new(AeMdw.Node, ...)` call on startup, I'm assuming it's just for documentation purposes?

Reply:
not needed anymore for dialyzer, thanks